app/models/conference_preference.rb

```diff
@@ -1,5 +1,21 @@
 class ConferencePreference < ActiveRecord::Base
+  validates :terms_accepted?, acceptance: true
```

Comment:
"only on create" related to the `ConferencePreference` record, not the `Team`. So it would be checked once, when the team first choses conferences.